Assets/_Scripts/Enemy/EnemyManager.cs

```diff
@@ -209,4 +214,11 @@ private bool IsAllEnemiesKilled()
     {
         return _enemiesSpawned == totalEnemies && _enemiesKilled == totalEnemies;
     }
+
+    // Adds to _enemiesKilled and subtracts from _enemiesAlive when EnemyKilled event is invoked.
+    private void TotalNumberOfEnemiesKilled()

```

Comment:
nit - it's good practice for function names to clearly state what the function is doing. In this case the function is marking statistics for when an enemy is killed. So something like `MarkEnemyKilledStats` would be better for a function name